packages/bsync/proto/bsync.proto

```diff
@@ -66,7 +68,45 @@ message ScanNotifOperationsResponse {
   string cursor = 2;
 }
 
+message AddPurchaseOperationRequest {
+  string actor_did = 1;
+}
+
+message AddPurchaseOperationResponse {}
+
+message Subscription {
+  string status = 1;
+  string renewalStatus = 2;
+  string group = 3;
+  string platform = 4;
+  string offering = 5;
+  google.protobuf.Timestamp periodEndsAt = 6;
+  google.protobuf.Timestamp periodStartsAt = 7;
+  google.protobuf.Timestamp purchasedAt = 8;
+}
+
+message GetSubscriptionsRequest {
```

Comment:
I forget, is there as reason we use "subscription" here but "add" in other locations?